app/views/layouts/application.html.erb

```diff
@@ -1,17 +1,31 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>.gov.au Transitions Beta</title>
+    <title>GOV.AU Transitions</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <%= csrf_meta_tags %>
+    <%= favicon_link_tag %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js"></script>
+    <script>
+      WebFont.load({
```

Comment:
This also causes the font flash on the page. this was a temp fix from the ui kit people